### PR TITLE
remove_unused_properties_part4

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

I have removed five groups of unused properties, and they are "asset.categories.properties.default", "Zip.file.name.encoding", "My.sites.display.style", "Layout.comments.enabled", "Rss.publish.to.live.by.default", "Calendar.publish.to.live.by.default".

asset.categories.properties.default - https://github.com/tinatian/liferay-portal/pull/2506
zip.file.name.encoding - https://github.com/tinatian/liferay-portal/pull/2496
my.sites.display.style - https://github.com/tinatian/liferay-portal/pull/2497
layout.comments.enabled - https://github.com/tinatian/liferay-portal/pull/2506
rss.publish.to.live.by.default - https://github.com/liferay-core-infra/liferay-portal/pull/234
calendar.publish.to.live.by.default - https://github.com/tinatian/liferay-portal/pull/2492
Thanks for checking. can also see https://github.com/jeffwu0724/liferay-portal/pull/26

Thanks,
Jeff

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test

**Comment by jeffwu0724** [comment]
@kyle-miho Hey Kyle, could you please help me to scan the test result?

Thanks,
Jeff

**Comment by kyle-miho** [comment]
@jeffwu0724 failures unrelated

File: portal-impl/src/com/liferay/portal/verify/VerifyProperties.java [hunk 1]
```diff
@@ -1744,7 +1746,7 @@ protected void verifySystemProperties() throws Exception {
 		"jpa.provider", "jpa.provider.property.eclipselink.allow-zero-id",
 		"jpa.provider.property.eclipselink.logging.level",
 		"jpa.provider.property.eclipselink.logging.timestamp",
-		"language.display.style.options",
+		"language.display.style.options", "Layout.comments.enabled",
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/verify/VerifyProperties.java:1749]: @jeffwu0724 The first character should be lower cased.

File: portal-impl/src/com/liferay/portal/verify/VerifyProperties.java [hunk 2]
```diff
@@ -1580,7 +1581,8 @@ protected void verifySystemProperties() throws Exception {
 		"browser.compatibility.ie.versions",
 		"buffered.increment.parallel.queue.size",
 		"buffered.increment.serial.queue.size",
-		"cache.clear.on.context.initialization", "captcha.max.challenges",
+		"cache.clear.on.context.initialization",
+		"Calendar.publish.to.live.by.default", "captcha.max.challenges",
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/verify/VerifyProperties.java:1585]: @jeffwu0724 The first character should be lower cased.

**Comment by jeffwu0724** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portal/verify/VerifyProperties.java (hunk 2 above)
Comment (jeffwu0724) [reply portal-impl/src/com/liferay/portal/verify/VerifyProperties.java]: Hey Tina, sorry for the mistakes. Already fixed

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/114170

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**